app/models/profile.rb

```diff
@@ -33,6 +33,7 @@ def decrypt_pii(user_access_key)
   end
 
   def recover_pii(personal_key)
+    personal_key = personal_key.tr('-', ' ')
```

Comment:
So maybe we should make `PersonalKeyGenerator.normalized_code` into a public method and rename it `normalize` -- that way we can consistently turn user input into something we can use to decipher.